_changelog/entries/2021/2021-03-12-pendo-v1-ijson-performance-enhancement.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Pendo (v1) improvement: using ijson to improve `visitors` and `events` replication
+content-type: "changelog-entry"
+date: 2021-03-12
+entry-type: improvement
+entry-category: "integration"
+connection-id: "pendo"
+connection-version: "1"
+---
+
+{{ site.data.changelog.metadata.single-integration | flatify }}
+
+ijson is now being used for {{ this-connection.display_name }}'s `visitors` and `events` streams to improve performance. It reduces the amount of memory needed to replicate its potentially large amounts of data.
```

Comment:
```suggestion
We've implemented [ijson](https://github.com/ICRAR/ijson), which is used in other Singer taps, to streamline replication for {{ this-connection.display_name }}'s `visitors` and `events` tables. This improvement reduces the amount of memory needed to replicate large amounts of data, potentially leading to improved Extraction times.
```

I made this a bit friendlier-sounding and tried to convey why this change is good for the end user. I like it when changelogs include pronouns and personality and not just technical mumbo jumbo - it reminds the end users that there are people working on the product. I see it as us talking to them every week and chatting about what we've been doing.